Sources/MongoSwift/BSON/BsonValue.swift

```diff
@@ -423,7 +423,7 @@ extension Int64: BsonValue {
 }
 
 /// A struct to represent the BSON Code and CodeWithScope types.
-public struct CodeWithScope: BsonValue, Decodable, Equatable {
+public struct CodeWithScope: BsonValue, Codable, Equatable {
```

Comment:
nit: everything else seems to be using the form: `BsonValue, Equatable, Codable`

Reply:
fixed